### PR TITLE
Production update files

### PR DESCRIPTION
**Pull request by samleeflang**
- Route changes (some need application changes which will also result in test/acc changes)
- Back to login-demo
- Added specific SA and SA tokens
- Terms pages online
- Schema to procution
- Update Kafka
- Couple other small things

### CODE REVIEW

File: production/data-exporter-backend/data-exporter-backend-route.yaml [hunk 1]
```diff
@@ -18,7 +18,15 @@ spec:
   entryPoints:
     - websecure
   routes:
-    - match: Host(`dissco.eu`) && PathPrefix(`/api/v1/data-export`)
+    - match: Host(`disscover.dissco.eu`) && PathPrefix(`api/v1/data-export`)
```
Comment (southeo) [inline production/data-exporter-backend/data-exporter-backend-route.yaml:21]: no `api.dissco.eu` routing for the data exporter backend?

File: production/backend/backend-route.yaml [hunk 2]
```diff
@@ -18,9 +18,61 @@ spec:
   entryPoints:
     - websecure
   routes:
-    - match: Host(`api.dissco.eu`)
+    - match: Host(`disscover.dissco.eu`) && PathPrefix(`/api/v1`)
```
Comment (southeo) [inline production/backend/backend-route.yaml:21]: So this first routing maps `disscover.dissco.eu/api/v1` -> `api.dissco.eu` Then the rest of the routings map `api.dissco.eu/v1/{endpoint}` back to the backend? 

Do you need all the routings? 

Also, can you map `disscover.dissco.eu/api/v1` to `api.disscover.eu/v1` instead? that way the other mappings won't need the `/v1` in the routing

File: production/orchestration-backend/orchestration-backend.yaml [hunk 3]
```diff
@@ -47,22 +47,22 @@ spec:
             - name: mas.kafkaHost
               value: kafka-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092
             - name: token.id
-              value: dissco-orchestration
+              value: production-dissco-orchestration
```
Comment (southeo) [inline production/orchestration-backend/orchestration-backend.yaml:50]: Do you also need to update the token id for the orchestration front end?

You mentioned you would call it `production-orchestration`, I actually think that is a bit unclear, adding `-fe` would be better if possible.

**Review by southeo — APPROVED** [review]
:traffic_light: 
Great to have these routes set up! few comments

File: production/data-exporter-backend/data-exporter-backend-route.yaml (hunk 1 above)
Comment (samleeflang) [reply production/data-exporter-backend/data-exporter-backend-route.yaml:21]: Yeah it is there `api.dissco.eu/v1/data-export` but that will change to `api.dissco.eu/data-export/v1`

File: production/backend/backend-route.yaml (hunk 2 above)
Comment (samleeflang) [reply production/backend/backend-route.yaml:21]: Not sure what you mean but we need the routes as `api.disscover.eu/v1` is also used for the other backends so we need to have the specific resource in there

File: production/orchestration-backend/orchestration-backend.yaml (hunk 3 above)
Comment (samleeflang) [reply production/orchestration-backend/orchestration-backend.yaml:50]: Yes Tom is checking how we can inject the FE token so we can have different tokens for prod/acc/test. I will suffic those with `-fe`

File: production/data-exporter-backend/data-exporter-backend-route.yaml (hunk 1 above)
Comment (samleeflang) [reply production/data-exporter-backend/data-exporter-backend-route.yaml:21]: Going to redo the routes with the new setup. Merging this one so the routes will get their own PR

**Pull request merged**